client/package.json

```diff
@@ -25,7 +25,8 @@
     "extends": "./configuration/eslint/eslint.js"
   },
   "eslintIgnore": [
-    "src/util/stateManager/matrix_generated.js"
+    "src/util/stateManager/matrix_generated.js",
```

Comment:
this file is no longer machine generated, so it should not be ignored by the linter